azure-pipelines.yml

```diff
@@ -233,7 +233,10 @@ jobs:
         ConfigOpts: --security --features=versioned_namespace=1
         PackageDeps: libxerces-c-dev libssl-dev cmake
       SecurityWithoutFeatures:
-        ConfigOpts: --no-inline --no-debug --no-built-in-topics --no-content-subscription --no-ownership-profile --no-object-model-profile --no-persistence-profile --security
+        ConfigOpts: >
+          --security --no-inline --no-debug
+          --no-built-in-topics --no-content-subscription --no-ownership-profile
```

Comment:
I don't think that we should be using --no-built-in-topics as those are needed for dynamic discovery aspects. Has there been prior discussion to drop them out for some reason?

Reply:
That's not part of Jeremy's changes. Jeremy - this branch now contains rewritten commits, please avoid this.  Merge again and force-push to reset your branch.

Reply:
Understood ... ignore my question.

Reply:
Gotcha. I didn't use a separate branch for this thinking it was a quick change.